autotest/gcore/tiff_read.py

```diff
@@ -3756,6 +3756,20 @@ def tiff_read_zstd_corrupted2():
     with gdaltest.error_handler():
         return ut.testOpen()
 
+
+###############################################################################
+# Test WEBP compression
+
+
+def tiff_read_webp():
+
+    md = gdal.GetDriverByName('GTiff').GetMetadata()
+    if md['DMD_CREATIONOPTIONLIST'].find('WEBP') == -1:
+        return 'skip'
+
+    ut = gdaltest.GDALTest('GTiff', 'tif_webp.tif', 1, None)
```

Comment:
Perhaps you could try to at least check the statistics so that pixel content is at least partly checked

Reply:
The stats change depending on the version of libwebp, the change is slight (for that file 216 vs 215) but enough that I couldn't test.

This is an internal version of libtiff, do I just copy over the files and submit a new PR to libtiff?

Reply:
I added - https://gitlab.com/libtiff/libtiff/merge_requests/32

Reply:
I was suggesting to use stats because you can compare got values with expected values with a tolerance margin, whereas GDAL checksum algorithm will change "randomly" even on small pixel value changes